bundles/org.openhab.binding.ahawastecollection/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -16,24 +15,24 @@
 		</channels>
 
 		<config-description>
-			<parameter name="commune" type="text" required="true">
+			<parameter name="commune" required="true" type="text">
```

Comment:
why have they been swapped around?

Reply:
It is (was) a "feature" of the XML DOM parser -- according to xml specifications the order of attributes is irrelevant.

.. HOWEVER .. I made a change to minimise the amount of 'tampering' with existing xml files.